### DISCUSSION TITLE
Phoenix desktop black screen on start up on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by gerd21** [phcode-dev/phoenix#1938] [Q&A] [answered]
Issue seems to be similar to [this](https://github.com/orgs/phcode-dev/discussions/1506).

I'm running Ubuntu 22.04 latest. Here I installed Phoenix automatically as proposed and installation ends without issues and the hint I can open files now.

If I do so I get a blank white desktop without anything in it. Playing around with the mouse I sometimes can see some code of the file, but it removes if I release the mouse.

So I installed Phoenix in a Virtual Machine in the same way and it works fine, as expected.

On my host system I'm using a Nvidia card. This card needs a driver named 'nvidia-driver-470'. I changed this proprietary driver to 'xserver-org-video-nouveau' and after reboot Phoenix shows everything as it should be.

Unfortunately my system really needs the nvidia driver, it runs unstable with the open source driver. So I have to switch back to nvidia and Phoenix doesn't work any more.

Starting Phoenix in terminal I get this:
<pre>$ phcode 

** (phoenix-code:34593): WARNING **: 11:27:52.363: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.

** (phoenix-code:34593): WARNING **: 11:27:52.367: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.

** (phoenix-code:34593): WARNING **: 11:27:52.371: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.
Appdata path is /home/gerhard/.local/share/io.phcode
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Keine Berechtigung
Failed to create GBM buffer of size 1366x900: Keine Berechtigung
</pre>
Any suggestion?

**Comment by abose** [comment, marked as answer]
Have you tried the fix listed in the linked issue?
add `WEBKIT_DISABLE_DMABUF_RENDERER=1` to `/etc/environment`

**Reply by gerd21** [reply]
Yes, I did. But it didn't help. 
**BUT**: After reboot everything seems to be perfect. 
Sorry, my fault.
Thx!!

**Reply by FernandoBasso** [reply]
I faced the same problem (on Arch Linux). As I didn't want to reboot, I was able to get it working by running `phcode` like this:

```bash
$ WEBKIT_DISABLE_DMABUF_RENDERER=1 phcode .
```
